src/index.ts

```diff
@@ -40,14 +40,9 @@ function setupOpenMRS() {
     extensions: [
       {
         id: 'registration-home-link',
-        slot: 'home-page-buttons',
+        slot: 'homepage-dashboard-slot',
         load: getAsyncLifecycle(() => import('./home-link'), options),
       },
-      {
-        id: 'registration-nav-link',
-        slot: 'nav-menu',
-        load: getAsyncLifecycle(() => import('./nav-link'), options),
-      },
       {
         id: 'patient-photo-widget',
         slot: 'patient-photo',
```

Comment:
Should that be `patient-photo-slot`?

Reply:
Yes it should be that, i have updated the name